packages/drupal/writer.js

```diff
@@ -6,13 +6,19 @@ const { isEqual } = require('lodash');
 const { drupalToCardstackDoc, cardstackToDrupalDoc } = require('./lib/document');
 const pendingChanges = new WeakMap();
 const log = require('@cardstack/logger')('cardstack/drupal');
+const { declareInjections } = require('@cardstack/di');
```

Comment:
I'm fine with deleting modules we aren't actively using like `@cardstack/drupal`. We could always resurrect them out of source control next time we actually want them.